### PR TITLE
Add reset

### PR DESCRIPTION
**Pull request by istarkov**
https://github.com/renatorib/react-powerplug/issues/110

### CODE REVIEW

File: src/index.js.flow [hunk 1]
```diff
@@ -269,6 +276,7 @@ type ValueChange<T> = (value: T) => void
 type ValueRender<T> = ({|
   value: T,
   set: Updater<T>,
+  reset: (cb?: () => void) => void,
```
Comment (TrySound) [inline src/index.js.flow]: Move please reset to separate type like `Updater`

File: src/index.js.flow [hunk 2]
```diff
@@ -5,7 +5,7 @@ import * as React from 'react'
 /* Utils */
 
 type Updater<T> = (updater: (T => T) | T) => void
-
+type Reset = (cb?: () => void) => void
```
Comment (TrySound) [inline src/index.js.flow]: add new line please

**Comment by TrySound** [comment]
Would be good to have `reset` in `Input` component too.

**Comment by renatorib** [comment]
Sorry for delay =P
Thanks so much guys <3

**Pull request merged**